source/extensions/filters/http/jwt_authn/jwks_cache.cc

```diff
@@ -43,6 +43,9 @@ class JwksDataImpl : public JwksCache::JwksData, public Logger::Loggable<Logger:
 
     tls_.set([](Envoy::Event::Dispatcher&) { return std::make_shared<ThreadLocalCache>(); });
 
+    tls_->jwt_cache_ = JwtCache::create(jwt_provider.enable_jwt_cache(),
```

Comment:
Since jwt_cache is inside ThreadLocalCache,  you have to create it in the lambda function in tls_set() call.

pass (bool jwt_enable_cache, int jwt_cache_size, TimeSource&) to ThreadLocalCache,  in its constructor calls JwtCache::create

Reply:
Done